### ISSUE TITLE
After successful reconnection, "Connection lost" is still displayed

### ISSUE THREAD

**Issue by nokhub** [tutao/tutanota#4663]
**Bug in native offline mode**

**Describe the bug**
While on any native client and logging into offline mode, the mail viewer still displays the "Connection lost" message while trying to load new emails, even if the client successfully reconnected, as indicated by the status in the top bar. This will result in no more emails being loaded until the "Try again" button next to the "Connection lost" is manually clicked.

**To Reproduce**
1. use either native desktop or android client
2. Use a premium account with enough emails in the inbox e.g. bedpremium
3. Log out
4. turn off device internet
5. Log in to the offline mode of the premium account
6. scroll down in the mail
7. see both, offline status in the top bar and "Connection lost" note at the bottom of the inbox
8. turn on device internet again
9. wait for the client to reconnect or press reconnect in the top bar
10. "Connection lost" note is still at the bottom of the mailbox and no more emails are loaded

**Expected behavior**
The connection lost note should disappear and the missing emails should be automatically loaded

**Screenshots**
before reconnecting in offline mode:
![reconnect1](https://user-images.githubusercontent.com/115003415/195357145-1252958f-b423-4d54-abc3-20290063a82d.png)

after reconnecting:
![reconnect2](https://user-images.githubusercontent.com/115003415/195357172-68e0c26b-b0f3-44a4-bb32-05e1573f2857.png)

**Comment by charlag** [comment]
I thought this happens for non-offline login too, I think this is expected for offline login? Why should we load the list automatically?
@nokhub @bedhub

**Comment by nokhub** [comment]
The behavior is right but the label "Connection lost" is misleading. For now it should be sufficient to change the labels text.

**Comment by vitoreiji** [comment]
Users have also complained about the message "looks like you are offline", when they can connect to other services, just not Tutanota, e.g. when we have downtime. Maybe rephrase that as well.

**Comment by bedhub** [comment]
As discussed in the refinement session from 04.11.2022 we agreed that the "Connection lost" message within the list is confusing and  we want to remove the text completely.

We accept that we do not automatically load the rest of the email in all cases. We will just have centered button there with the text "Load more".
